### DISCUSSION TITLE
Carlton has started a open source maintainer mentorship group!

### DISCUSSION THREAD

**Discussion by tim-schilling** [django-commons/membership#38] [Show and tell]
His :tada: announcement blog post can be found here: https://noumenal.es/posts/mentoring-maintainers/6WB/ 
His open source maintainer discussion group can be found here: https://github.com/carltongibson/notes/discussions

**Comment by carltongibson** [comment]
Hey @tim-schilling — yep, thanks for mentioning it. All are welcome. 

Starting slowly (obviously) — do say hello! 

There's already some good comments, so I'll use some of those as inspiration for small posts, hopefully then sparking more discussion.

**Reply by tim-schilling** [reply]
That makes a lot of sense. I wanted to highlight things here in case someone stopped by and didn't know about the other resources available.
